src/DataBaseHelper.java

```diff
@@ -0,0 +1,39 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+/* Here I am storing all the user given data and results along with data and time of the operation into database*/
+class DataBaseHelper {
+
+    protected DataBaseHelper(String word, String filepath, String resultToDatabase, int totalNoOfWords, String errorMessage) throws SQLException {
+        storingDataToDataBase(word, filepath, resultToDatabase, totalNoOfWords, errorMessage);
+    }
+
+    private static void storingDataToDataBase(String word, String filepath, String resultToDatabase, int totalNoOfWords, String errorMessage) throws SQLException {
+        Connection connectionToDataBase = null;
+        String driveClass = "com.mysql.cj.jdbc.Driver";
+        String mySqlUrl = "jdbc:mysql://localhost:3306/elixr1";
+        String userName = "root";
+        String passwordOfDatabase = "Ashu@2406";
+        String dateAndTimeFormat = "yyyy/MM/dd HH:mm:ss";
+
+        try {
+            DateTimeFormatter dateAndTimeFormater = DateTimeFormatter.ofPattern(dateAndTimeFormat);
+            LocalDateTime now = LocalDateTime.now();
+            String currentDateAndTime = dateAndTimeFormater.format(now);         //current date and time.
+            Class.forName(driveClass);
+            connectionToDataBase = DriverManager.getConnection(mySqlUrl, userName, passwordOfDatabase);
+            Statement st = connectionToDataBase.createStatement();
```

Comment:
All the lines from 28 -30 should be in a separate method that returns connection object and just call that method whenever you want to create statements or preparedstatements to execute queries. This makes connection object re-usable .For a single usecase what you did is fine but tommorow if you want to run a select query from the same table, then you dont have to repeat lines

Reply:
Ok sure I will resolve those things.